package.json

```diff
@@ -33,6 +34,7 @@
     "@types/jest": "29.2.4",
     "babel-loader": "9.1.0",
     "css-loader": "6.7.2",
+    "figma-api-exporter": "github:slawomirkolodziej/figma-api-exporter#664caaa010f295bde15ec4db752abd0a2cc8d2af",
```

Comment:
Do we need a sec review on this one?

Reply:
We do indeed - it's over here https://github.com/brave/security/issues/1137
Includes the dependency & the `@html` review :smile: